CHANGELOG.md

```diff
@@ -6,11 +6,17 @@
 
 ## Changed
 
+# 0.0.5
```

Comment:
```suggestion
```

You don't need to manually add the version string, our release tooling does that automatically.

Reply:
Thanks, I thought I was doing it wrong. I think I've got it right this time (I had "## Fixed" twice after removing the version string so also removed that)?

Reply:
All you need to do is add a bullet with your changes, so it's correct now.